Ports/.port_include.sh

```diff
@@ -747,64 +747,13 @@ do_dev() {
-                    if git apply < $patch; then
-                        git add -A
-                        if prompt_yes_no "- This patch does not appear to be a git patch, would you like to modify its changes before continuing?"; then
-                            >&2 echo "Apply any changes you want, commit them into the current repo and quit this shell to continue."
-
-                            launch_user_shell
-                        fi
-                        main_author=''
-                        co_authors=()
-                        while read -r line; do
-                            author="$(echo "$line" | cut -f2 -d'	')"
-                            if [[ -z "$main_author" ]]; then
-                                main_author="$author"
-                            else
-                                co_authors+=("$author")
-                            fi
-                        done < <(git -C "${PORT_META_DIR}" shortlog -esn -- "patches/$(basename "$patch")")
-
-                        if [[ -n "$main_author" ]]; then
-                            date="$(git -C "${PORT_META_DIR}" log --format=%ad -n1 -- "patches/$(basename "$patch")")"
-                            >&2 echo -n "- This patch was authored by $main_author"
-                            if [[ ${#co_authors[@]} -ne 0 ]]; then
-                                >&2 echo -n " (and ${co_authors[*]})"
-                            fi
-                            >&2 echo " at $date"
-                            if prompt_yes_no_default_yes "- Would you like to preserve that information?"; then
-                                trailers=()
-                                for a in "${co_authors[@]}"; do
-                                    trailers+=("--trailer" "Co-Authored-By: $a")
-                                done
-                                git commit --verbose --author "$main_author" --date "$date" "${trailers[@]}"
-                            else
-                                >&2 echo " Okay, using your current git identity as the author."
-                                git commit --verbose
-                            fi
-                        else
-                            git commit --verbose
-                        fi
-                    else
-                        # The patch didn't apply, oh no!
-                        # Ask the user to figure it out :shrug:
-                        git am --keep-cr --keep-non-patch --3way "$patch" || true
-                        >&2 echo "- This patch does not apply, you'll be dropped into a shell to investigate and fix this, quit the shell when the problem is resolved."
-                        >&2 echo "Note that the patch needs to be committed into the current repository!"
-                        launch_user_shell
-                        force_patch_regeneration='true'
-                    fi
+                if ! git am --keep-cr --keep-non-patch --3way "$patch"; then
```

Comment:
:yakkiss:

Reply:
thanks github, I totally meant to submit this as a review 10/10 "cancel" feature.